src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int MAX_VALUE = 5;
+    private static final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static Figure getDefaultFigure() {
+        return new Circle("White", 10);
+    }
+
+    public static Figure getRandomFigure() {
+        int figureType = random.nextInt(MAX_VALUE);
+        String color = ColorSupplier.getRandomColor();
```

Comment:
Use the instance of 'ColorSupplier' that you have created in the class ('colorSupplier') instead of calling a static method on the 'ColorSupplier' class.